src/Google.Api.Gax/FixedSizePage.cs

```diff
@@ -19,7 +19,14 @@ public sealed class FixedSizePage<TResource> : IEnumerable<TResource>
     {
```

Comment:
But the method that returns this is currently called GetCompletePage(). Having the two match seems like a good idea.